### ISSUE TITLE
Save user id instead 

### ISSUE THREAD

**Issue by fabri2000779** [ass-a-service/ajobot#25]
Right now we save username like fabri2000779#3306 but in discord you can change #<number> if you pay. This cause if a user gets Discord Nitro and change his #<number> then he will lost all his ajos.

The solution for this is save the discord user id that never changes

**Comment by axl89** [comment]
Proposal: change the `lb` sorted set from:

```
127.0.0.1:6379> zrange lb 0 -1 withscores
 1) "pepahell#2102"
 2) "0"
 3) "Ajo bot#9944"
 4) "1"
...
```

to:

```
127.0.0.1:6379> zrange lb 0 -1 withscores
 1) 458641181818
 2) "0"
 3) 48412418232
 4) "1"
...
```

The conns I can think of:
- Leaderboard command would require to call discord 10 times (since we only print 10 entries) to translate Discord User IDs to User names

**Comment by Zymna** [comment]
Fixed with #26
